src/Lib/Managers/ItemsManager.vala

```diff
@@ -110,6 +113,11 @@ public class Akira.Lib.Managers.ItemsManager : Object {
         items.append (item);
     }
 
+    public void on_request_delete_item (Lib.Models.CanvasItem item_to_delete) {
+        item_to_delete.delete ();
+        canvas.window.event_bus.item_deleted (item_to_delete);
+    }
```

Comment:
Same thing here for the variable name, just call it `item`.

Reply:
Sure, as before!